### PR TITLE
Update for 1.0.0-alpha compatibility

### PR DESCRIPTION
**Pull request by sbward**
* Tested with `--cfg target_word_size=\"32\"` and `--cfg target_word_size=\"64\"`
* Fixes several warnings, but "unstable" and "Copy impl" warnings are still there

### CODE REVIEW

File: src/appkit.rs [hunk 1]
```diff
@@ -261,34 +261,34 @@ pub enum NSEventType {
 
 #[repr(u64)]
```
Comment (bjz) [inline src/appkit.rs]: should this be `u32`?

**Comment by bjz** [comment]
Could you double check the size of the enum descriminants for me?

**Comment by sbward** [comment]
@bjz I'm not an expert with Cocoa internals but it seems a 32 bit enum fits this number of flags and it compiled, so I've included that change

**Comment by bobakker** [comment]
Can we please merge this now? The library not compiling on both the alpha and the nightlies seems like a big deal to me...

**Pull request merged**